silver/tests/spec/test_payment_method.py

```diff
@@ -1,49 +1,330 @@
+import sys
+from copy import deepcopy
+
+from rest_framework import permissions, status
 from rest_framework.reverse import reverse
+from six import iteritems
 
 from silver.api.serializers import PaymentMethodSerializer
 from silver.models import PaymentMethod
+from silver.api.views import PaymentMethodList, PaymentMethodDetail
 from silver.tests.factories import CustomerFactory, PaymentMethodFactory
+from silver.tests.spec.test_transactions import register
 from silver.tests.spec.util.api_get_assert import APIGetAssert
 
 
 class TestPaymentMethodEndpoints(APIGetAssert):
     serializer_class = PaymentMethodSerializer
 
-    def test_filter_processor(self):
-        customer = CustomerFactory.create()
-        payment_method = PaymentMethodFactory.create(
-            customer=customer
-        )
+    def setUp(self):
+        self.customer = CustomerFactory.create()
+
+        super(TestPaymentMethodEndpoints, self).setUp()
+
+    def create_payment_method(self, *args, **kwargs):
+        method = PaymentMethodFactory.create(*args, **kwargs)
 
         # mysql does not store fractional time units but the object
         # created will have them so we can't use it directly
         #  to check the output
-        payment_method.refresh_from_db()
+        method.refresh_from_db()
```

Comment:
And honestly parsing datetime to convert and stuff would end up in more lines of code, One solution would be to force django to store microseconds in mysql but then we would need to check if mysql can take that route as not all versions can...